src/extension/ncwms/src/test/java/au/org/emii/geoserver/wms/LayerDescriptorTest.java

```diff
@@ -0,0 +1,25 @@
+package au.org.emii.geoserver.wms;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class LayerDescriptorTest extends TestCase {
+    @Test
+    public void testSimpleParsing() throws Exception {
+        LayerDescriptor layerDescriptor;
+
+        layerDescriptor = new LayerDescriptor("imos:some_layer/sea_water_velocity");
+        assertEquals("imos", layerDescriptor.workspace);
+        assertEquals("some_layer", layerDescriptor.layer);
```

Comment:
`some_layer` is actually a group layer in this case
`sea_water_velocity` is actually a layer name. Assigning it to `variable` in this code simply looks weird.  We are interested in what is the layer name.